src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -102,15 +102,14 @@ def load_arguments(self, _):
         c.argument('resource_group_name', arg_type=resource_group_name_type, help='the resource group where the policy will be applied')
 
     with self.argument_context('policy definition', resource_type=ResourceType.MGMT_RESOURCE_POLICY) as c:
-        from azure.mgmt.resource.policy.models import PolicyMode
         c.argument('policy_definition_name', arg_type=existing_policy_definition_name_type)
         c.argument('rules', help='JSON formatted string or a path to a file with such content', type=file_type, completer=FilesCompleter())
         c.argument('display_name', help='Display name of policy definition.')
         c.argument('description', help='Description of policy definition.')
         c.argument('params', help='JSON formatted string or a path to a file or uri with parameter definitions.', type=file_type, completer=FilesCompleter(), min_api='2016-12-01')
         c.argument('metadata', min_api='2017-06-01-preview', nargs='+', validator=validate_metadata, help='Metadata in space-separated key=value pairs.')
         c.argument('management_group', arg_type=management_group_name_type)
-        c.argument('mode', arg_type=get_enum_type(PolicyMode), options_list=['--mode', '-m'], help='Mode of the policy definition.', min_api='2016-12-01')
+        c.argument('mode', options_list=['--mode', '-m'], help='Mode of the policy definition, e.g. All, Indexed. Please visit https://aka.ms/azure-policy-mode for more information.', min_api='2016-12-01')
```

Comment:
I think something like this would be more helpful: 'Mode of the policy definition. Values: one of All, Indexed, or <resource-provider>. See https://aka.ms/azure-policy-mode for more information.' The link already mentions resource provider modes, and will be updated as new ones are added. If you explicitly want to leave out <resource-provider> from that list, then I'd suggest wording like this: 'Mode of the policy definition, usually All or Indexed. See https://aka.ms/azure-policy-mode for more information.'